shopify_python/git_utils.py

```diff
@@ -60,6 +59,20 @@ def changed_python_files_in_tree(root_path):
             if os.path.exists(abs_mod) and os.path.isfile(abs_mod) and _file_is_python(abs_mod)]
 
 
+def uncommitted_python_files(root_path):
```

Comment:
Let's add a docstring to both of these methods to explain what they do.

Reply:
Also, why group unstaged and staged files in one method, and keep untracked separate?

Perhaps it would be better to split these into two, and then add one more method to union them, something like this:

- `untracked_python_files `
- `untstaged_python_files `
- `staged_python_files `
- `changed_local_python_files` (union the three above)